README.md

```diff
@@ -18,74 +16,84 @@ This library is part of [ACPaaS UI][acpaas-ui].
 | Schematics        | ES5+       | [![acpaas-ui-schematics-github]][acpaas-ui-schematics] |
 | Javascript        | ES5+       | [![acpaas-ui-js-github]][acpaas-ui-js] |
 | Angular           | Angular 6+ | [![acpaas-ui-angular-github]][acpaas-ui-angular] |
-| React             | React 16+  | [acpaas-ui-react] |
-| VueJS             | TBA        | TBA  |
+| React             | React 16+  | [![acpaas-ui-react-github]][acpaas-ui-react] |
+
+## Getting Started
+
+Start a new Angular app.
+
+```sh
+npm install -g @angular/cli
+ng new my-app --style=scss
+cd my-app
+npm install @a-ui/core @acpaas-ui/ngx-components
+```
+
+Edit styles.scss to import the branding look & feel.
+
+```scss
+/* core branding, https://a-ui.github.io/core_branding_scss/ */
+@import '@a-ui/core/dist/main.css';
```

Comment:
Import core_branding only for variables / functions / mixins / ... the main styling should be loaded through the CDN as described on: https://github.com/a-ui/core_branding_scss unless there's a specific project-related reason not to do it.

Reply:
I would also prefer "the CDN way" to be documented here and maybe link to the core branding for more advanced setups.

Reply:
changed it to the CDN way